clang-format.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+
+CLANG_FORMAT_VERSION="clang-format"
```

Comment:
Why the _VERSION suffix?

Reply:
ah, because I copy-pasted from [check-clang-format.sh](https://github.com/open-telemetry/opentelemetry-network/blob/7e1b5c7edfd14aa0f57ee235065ff51878d0c485/.github/workflows/scripts/check-clang-format.sh#L6). I'll fix in a new PR